internal/inspect/report.go

```diff
@@ -44,15 +48,19 @@ func Report(ctx context.Context, out string, config pgconn.Config, fsys afero.Fs
 			return errors.Errorf("failed to read query: %w", err)
 		}
 		name := strings.Split(d.Name(), ".")[0]
-		outPath := filepath.Join(out, fmt.Sprintf("%s_%s.csv", name, date))
+		outPath := filepath.Join(dateDir, fmt.Sprintf("%s.csv", name))
 		return copyToCSV(ctx, string(query), outPath, conn.PgConn(), fsys)
 	}); err != nil {
 		return err
 	}
-	if !filepath.IsAbs(out) {
-		out, _ = filepath.Abs(out)
+	// print the actual save location
+	if !filepath.IsAbs(dateDir) {
+		dateDir, err = filepath.Abs(dateDir)
+		if err != nil {
+			return err
+		}
```

Comment:
Due to the way we change the workdir, we must use `utils.CurrentDirAbs` to handle absolute path.